.gitignore

```diff
@@ -0,0 +1,5 @@
+/test.md
+
+/node_modules
+
+/*.json
```

Comment:
- [ ] Kindly remove `/*.json` from `.gitignore` file. It is not part of the requirements.
- [ ] Please correct the name of `/node_modules` by `node_modules/`

![image](https://user-images.githubusercontent.com/58642949/191263442-7f48ad0e-29bc-4c4e-af7e-ce9b2e8a4895.png)